javav2/example_code/timestream/src/test/java/TimestreamTest.java

```diff
@@ -0,0 +1,151 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+import com.timestream.write.*;
+import com.timestream.query.*;
+import org.junit.jupiter.api.*;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.timestreamquery.TimestreamQueryClient;
+import software.amazon.awssdk.services.timestreamwrite.TimestreamWriteClient;
+import java.io.*;
+import java.util.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class TimestreamTest {
+
+    private static  TimestreamWriteClient timestreamWriteClient;
+    private static TimestreamQueryClient queryClient;
+    private static String dbName = "";
+    private static String newTable = "";
+
+    // TODO" Change database name
+    private static String queryString = "SELECT\n" +
+            "    truck_id,\n" +
+            "    fleet,\n" +
+            "    fuel_capacity,\n" +
+            "    model,\n" +
+            "    load_capacity,\n" +
+            "    make,\n" +
+            "    measure_name\n" +
+            "FROM \"ScottTimeDB\".IoTMulti" ;
```

Comment:
should change "ScottTimeDB" to something more generic and put it either in a config file or variable so it's not hard-coded in the middle of a query string.

Reply:
fixed

Reply:
fixed